backend/app/routers/visualization.py

```diff
@@ -172,23 +171,29 @@ async def save_visualization_config(
 
 @router.get("/{resource_id}/config", response_model=List[VisualizationConfigOut])
 async def get_resource_visconfig(
-    resource_id: PydanticObjectId,
-    user=Depends(get_current_user),
+        resource_id: PydanticObjectId,
+        user=Depends(get_current_user),
 ):
     query = [VisualizationConfigDB.resource.resource_id == ObjectId(resource_id)]
     visconfigs = []
     async for vzconfig in VisualizationConfigDB.find(*query):
-        visconfigs.append(vzconfig)
+        config_visdata = []
+        visdata_query = [VisualizationDataDB.visualization_config_id == vzconfig.id]
+        async for vis_data in VisualizationDataDB.find(*visdata_query):
+            config_visdata.append(vis_data.dict())
+        visconfig_out = VisualizationConfigOut(**vzconfig.dict())
+        visconfig_out.visualization_data = config_visdata
+        visconfigs.append(visconfig_out)
```

Comment:
The attaching visdata model in config needed for this endpoint. This i the endpoint being used by frontend